Doc/whatsnew/3.9.rst

```diff
@@ -262,6 +262,8 @@ Changes in the Python API
 * The :mod:`venv` activation scripts no longer special-case when
   ``__VENV_PROMPT__`` is set to ``""``.
 
+* The :mod:`importlib` module now ignore the :envvar:`PYTHONCASEOK`
```

Comment:
```suggestion
* The :mod:`importlib` module now ignores the :envvar:`PYTHONCASEOK`
```

Reply:
> I have made the requested changes; please review again

Well, you didn't update this one. English is not my first language, so I'm not sure, but it should be "The module ignoreS" here, no? with an S.

Reply:
Yep, you're right regarding the English grammar here, @vstinner.